nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -902,6 +918,7 @@ in {
         in {
           wantedBy = [ "multi-user.target" ];
           before = [ "phpfpm-nextcloud.service" ];
+          after = optional mysqlLocal "mysql.service" ++ optional pgsqlLocal "postgresql.service";
```

Comment:
I think it's also safer to do the same for `requires`.

Reply:
Nice catch